test/test_helper.rb

```diff
@@ -3,13 +3,24 @@
 ENV['RAILS_ENV'] ||= 'test'
 
 require 'simplecov'
-require 'coveralls'
-Coveralls.wear!
+SimpleCov.start do
+  add_filter 'gemfiles'
+  add_filter 'test'
```

Comment:
If possible I like to only restrict certain files vs all of `test`, because simplecov will catch dead test code.